Makefile

```diff
@@ -163,6 +163,7 @@ test: flake8
 	@echo "MIN_INSTANCE_COUNT_LOW: 2" >> data.yml
 	@echo "SCHEDULE_SCALER_ENABLED: True" >> data.yml
 	@echo "SQS_QUEUE_PREFIX: test" >> data.yml
+	@echo "DEFAULT_CPU_PERCENTAGE_THRESHOLD: 60" >> data.yml
```

Comment:
Since this is the default threshold it should go in `generate-config` target so that it takes effect on all environments.